src/SFA.DAS.EmployerFinance/Models/LevyDeclarationSagaTaskType.cs

```diff
@@ -0,0 +1,8 @@
+namespace SFA.DAS.EmployerFinance.Models
+{
+    public enum LevyDeclarationSagaTaskType : short
+    {
+        ImportPayeSchemeLevyDeclarations = 0,
```

Comment:
We don't strictly need the initializers.

Reply:
I usually add them to enums that are persisted (e.g. `[dbo].[LevyDeclarationSagaTasks].[Type]`) to avoid issues if/when new values are added or existing values are reordered.

Reply:
We could add them when they're actually needed, but they're doing no harm 🙂.